### PR TITLE
chore: Update README.md (Add Local Installation Steps)

### LINKED ISSUE #3167: Basic installation steps missing in README
**Actual Behaviour**
Currently README.md does not have installation steps to setup the project locally . This can create trouble for beginners who are new to open source and trying to set up the project locally.

**Expected Behaviour**
Required installation steps must be added in README.md .

**Additional Context**
I will work on this issue

### PR DESCRIPTION
**Pull request by snitin315**
Fixes #3167 

**Changes:**

Added local installation steps in README.md

Screenshots of the change: 

Screenshot from 2019-12-31 23-02-19

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,20 @@ SUSI.AI is written in JavaScript with React. To get started with the code, follo
 * node --version >= 6
 * yarn --version >= 3
 
+## Local Installation
+### Steps
+* `git clone <repository-url>` this repository
```
Comment (akshatnitd) [inline README.md]: Write `where repository-url is the link to the forked repository.

File: README.md [hunk 2]
```diff
@@ -38,6 +38,20 @@ SUSI.AI is written in JavaScript with React. To get started with the code, follo
...
+* `cd susi.ai`
+
+**Note :** If you want to contribute, first fork the original repository and clone the forked repository into your local machine followed by ```cd``` into the directory
+```sh
+git clone https://github.com/USERNAME/susi.ai.git
+cd susi.ai
+```
+* Install all the dependecies with `npm install`
+* Start the server with `npm start`
+* Visit your app at [http://localhost:300](http://localhost:300).
```
Comment (akshatnitd) [inline README.md]: Default port is `3000`

File: README.md (hunk 1 above)
Comment (snitin315) [reply README.md]: Done

**Comment by snitin315** [comment]
@akshatnitd Requested changes done. Please review.

File: README.md [hunk 3]
```diff
@@ -38,6 +38,20 @@ SUSI.AI is written in JavaScript with React. To get started with the code, follo
 * node --version >= 6
 * yarn --version >= 3
 
+## Local Installation
+### Steps
+* `git clone <repository-url>` where repository-url is the link to the forked repository
```
Comment (akshatnitd) [inline README.md]: `git clone <repository-url>`, where `<repository-url>` is the link to the forked repository

Comment (snitin315) [reply README.md]: @akshatnitd Done.

**Pull request merged**